.github/workflows/deny.yml

```diff
@@ -0,0 +1,20 @@
+name: cargo-deny
+on: [push, pull_request]
+jobs:
+  cargo-deny:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        checks:
+          # - advisories
```

Comment:
Just curious why this is disabled.

Reply:
This is from the original staging branch.
Actually I hope to enable checking for license:)